uwh-refbox/src/main.rs

```diff
@@ -453,7 +490,39 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         penalty_add_layout.attach(&penalty_referee_timeout, 3, 9, 6, 2);
         penalty_add_layout.attach(&penalty_black_timeout, 9, 9, 3, 2);
 
+        //Setting up the selected buttons
+        let penalty_black_select_ = penalty_black_select.clone();
+        let penalty_white_select_ = penalty_white_select.clone();
+
+        /*
+
+        let penalty_1min_ = penalty_1min.clone();
+        let penalty_2min_ = penalty_2min.clone();
+        let penalty_5min_ = penalty_5min.clone();
+        let penalty_dismiss_ = penalty_dismiss.clone();
+
+        */
```

Comment:
remove

Reply:
that's just where I was when at when I pushed it, I can re-create easily if you want it deleted